.github/PULL_REQUEST_TEMPLTE/projectTemplate.md

```diff
@@ -0,0 +1,14 @@
+## Template Info
+- Name: <!-- replace this comment with the name -->
+- Link to the NuGet Package: <!-- replace this comment with the link to NuGet package-->
+- Underlying SPA framework: <!-- Examples are React, Vue, Angular, ... -->
+
+## Description
+
+<!-- Tell us why this package is important and  -->
+
+## References
+<!-- To keep the bar high for these submissions, we require at least three blog posts (from different sources) referencing the package -->
```

Comment:
After thinking some more about this, I'm not sure whether this is worth doing or not, as it may slow down adoption of new templates.

So we can, potentially, eliminate this requirement, taking the risk of potentially expanding this list with a lot of low-quality templates.
Would appreciate others' thoughts on this.